src/main/java/games/strategy/engine/data/GameDataMemento.java

```diff
@@ -0,0 +1,136 @@
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Maps;
+
+import games.strategy.util.Version;
+import games.strategy.util.memento.MementoExporter;
+import games.strategy.util.memento.MementoImportException;
+import games.strategy.util.memento.MementoImporter;
+import games.strategy.util.memento.PropertyBagMementoExporter;
+import games.strategy.util.memento.PropertyBagMementoImporter;
+
+/**
+ * Provides factory methods for creating objects that can import mementos to and export mementos from game data.
+ */
+public final class GameDataMemento {
+  @VisibleForTesting
+  static final String SCHEMA_ID = "application/x.triplea.game-data-memento";
+
+  /**
+   * An immutable collection of default memento export options.
+   *
+   * <p>
+   * The key is the option name. The value is the option value.
+   * </p>
+   */
+  public static final Map<ExportOptionName, Object> DEFAULT_EXPORT_OPTIONS_BY_NAME = newDefaultExportOptionsByName();
+
+  private GameDataMemento() {}
+
+  @VisibleForTesting
+  interface PropertyNames {
+    String NAME = "name";
+    String VERSION = "version";
+  }
+
+  private static Map<ExportOptionName, Object> newDefaultExportOptionsByName() {
+    return Maps.immutableEnumMap(ImmutableMap.of(
+        ExportOptionName.EXCLUDE_DELEGATES, false));
+  }
+
+  /**
+   * Creates a new game data memento exporter using the default export options.
+   *
+   * @return A new game data memento exporter; never {@code null}.
+   */
+  public static MementoExporter<GameData> newExporter() {
+    return newExporter(DEFAULT_EXPORT_OPTIONS_BY_NAME);
+  }
+
+  /**
+   * Creates a new game data memento exporter using the specified export options.
+   *
+   * @param optionsByName The memento export options; must not be {@code null}. The key is the option name. The value is
+   *        the option value.
+   *
+   * @return A new game data memento exporter; never {@code null}.
+   */
+  public static MementoExporter<GameData> newExporter(final Map<ExportOptionName, Object> optionsByName) {
+    checkNotNull(optionsByName);
+
+    return new PropertyBagMementoExporter<>(SCHEMA_ID, new ExportHandler(optionsByName));
+  }
+
+  /**
+   * Names the options that can be specified when creating a new game data memento exporter.
+   */
+  public enum ExportOptionName {
+    /**
+     * Indicates delegates should be excluded from the memento (value type: {@code Boolean}).
+     */
+    EXCLUDE_DELEGATES
+  }
+
+  private static final class ExportHandler implements PropertyBagMementoExporter.Handler<GameData> {
+    // TODO: add support for options
+    @SuppressWarnings("unused")
+    private final Map<ExportOptionName, Object> optionsByName;
+
+    ExportHandler(final Map<ExportOptionName, Object> optionsByName) {
+      this.optionsByName = new HashMap<>(optionsByName);
+    }
+
+    @Override
+    public void exportProperties(final GameData gameData, final Map<String, Object> propertiesByName) {
```

Comment:
Thinking out loud, but perhaps you would appreciate this comment. This seems like a good place for open/close, injection of which properties are included.

EG: add a param:
```
 public void exportProperties(final GameData gameData, final Map<String, Object> propertiesByName, List<Pair<PropertyName, Function<GameData,String>>> properties) {
  properties.forEach(property ->
     propertiesByName.put(property.left(), property.right().apply(gameData)));
}
```